src/components/PageMarker/index.jsx

```diff
@@ -0,0 +1,27 @@
+import "./styles.css";
+import CircleMarker from "./components/CircleMarker";
+
+const PageMarker = ({ numOfPages, atualPage }) => {
+  const renderMarkers = () => {
+    const markers = [];
+
+    for (let i = 0; i < numOfPages; i++) {
+      if (atualPage - 1 === i) {
+        markers.push(<CircleMarker isActive />);
+      } else {
+        markers.push(<CircleMarker />);
+      }
+    }
```

Comment:
Sugestão de usar outra estrutura de controle:
`for (let i = 0; i < numOfPages; i++) {
atualPage - 1 === i  ?  markers.push(<CircleMarker isActive />)  :         markers.push(<CircleMarker />);
}
`